src/panels/Menu/TaskBar/TaskBar.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Flex } from '@mantine/core';
+import { WindowLayoutOptions } from 'src/windowmanagement/WindowLayout/WindowLayout';
+
+import { menuItemsData } from '@/windowmanagement/data/MenuItems';
+
+import { TaskBarMenuButton } from './TaskBarMenuButton';
+
+interface Props {
+  addWindow: (component: React.JSX.Element, options: WindowLayoutOptions) => void;
+  visibleMenuItems: string[];
+}
+
+export function TaskBar({ addWindow, visibleMenuItems }: Props) {
+  const checkedMenuItems = menuItemsData.filter((item) => {
```

Comment:
rename checkedMenuItems to visibleTaskBarButtons or something? Took me a while to understad what checked menu items was.